docs/client_advanced.rst

```diff
@@ -451,6 +451,24 @@ In order to specify the nameservers to when resolving the hostnames,
     conn = aiohttp.TCPConnector(resolver=resolver)
 
 
+Changing source interface
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you wish to specify where the packets should originate from, you should
+specify the interface you wish to use.
+Raises :exc:`RuntimeError` if the OS does not support this feature.
+:class:`~aiohttp.TCPConnector`::
```

Comment:
```suggestion
specify the interface you wish to use when initializing a :class:`~aiohttp.TCPConnector` instance.

.. warning::
   This may cause a :exc:`RuntimeError` if the OS kernel does not support this feature.
   
Here's an example of how to do this::
```